src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -90,17 +90,7 @@ public function getCreatedTime()
     }
 
     /**
-     * Returns the time the album was updated.
-     *
-     * @return \DateTime|null
-     */
-    public function getUpdatedTime()
-    {
-        return $this->getField('updated_time');
-    }
-
-    /**
-     * Returns the description of the album.
+     * Returns the `description` (The description of the album) as string if present.
```

Comment:
I feel like we should normalize the docblocks on all the getter methods across the board.

I think they should just be the description returned from `metadata=1`.

So this:

    Returns the `description` (The description of the album) as string if present.

Would just be

    The description of the album.

A curious dev just has to look down 2 lines to see what datatype it's being returned as and 2 more lines to see the name of the field. So it'd be nice to not have to maintain all that stuff in both the docblocks and actual code to reduce complexity.